src/utils/repo/create-browser.js

```diff
@@ -0,0 +1,28 @@
+/* global self */
+'use strict'
+
+const IPFSRepo = require('ipfs-repo')
+const hat = require('hat')
+
+const idb = self.indexedDB ||
+  self.mozIndexedDB ||
+  self.webkitIndexedDB ||
+  self.msIndexedDB
+
+function createTempRepo (repoPath) {
+  repoPath = repoPath || '/ipfs-' + hat()
+
+  const repo = new IPFSRepo(repoPath)
+
+  repo.teardown = (done) => {
+    repo.close(() => {
+      idb.deleteDatabase(repoPath)
+      idb.deleteDatabase(repoPath + '/blocks')
+      done()
+    })
+  }
+
+  return repo
+}
```

Comment:
Also based on https://github.com/ipfs/js-ipfsd-ctl/pull/243#issuecomment-388975434, this is an antipattern. What this is creating is that IPFS might fail to close the repo and we are forcely closing it and deleting.

The only things you should need for tests are the `idb.deleteDatabase` and `rimraf` calls